distributor-api-v1/deprecations/README.md

```diff
@@ -0,0 +1,5 @@
+# Deprecations
+
```

Comment:
Nice. Thx.

I would just add a paragraph:
```
This page contains a list of past and future deprecations in the API. Deprecation information are present in individual pages. `Deprecation notice` is a date from which we do support both versions (previous/deprecated and next/living standard). `Removal date` is a date where we aim to remove the previous/deprecated variant of the API.
```

cc @vitasamek maybe you can come up with some better phrasing 👍